### DISCUSSION TITLE
Changes to capture_RPiHQ.cpp

### DISCUSSION THREAD

**Discussion by EricClaeys** [AndreasLMeg/allsky#5] [Q&A] [answered]
@AndreasLMeg,
Are you changes to capture_RPiHQ.cpp all related to improving exposures?
How well does that work?
Are there changes to other files that are needed to get this to work?

Thanks - Eric

**Comment by AndreasLMeg** [comment, marked as answer]
Hello Eric,

I have made a README_mode_mean.md - but only in german.
https://github.com/AndreasLMeg/allsky/blob/master/README%20mode%20mean.md

Main changes are in mode_RPiHQ_mean.cpp, RPiHQ_modeMean.h, config.sh.repo, capture_RPiHQ.cpp

But the files are under construction and the changes maybe Q&D

After my hardware project I hopefully will find the time to finish the work ;-)

btw: using C++ Object Oriented programming concepts would be nice - to implement new cameras or new features like "auto gain",...

Best regards
Andreas

**Comment by AndreasLMeg** [comment]
btw: using C++ Object Oriented programming concepts would be nice - to implement new cameras or new features like "auto gain",...
